.github/workflows/auto-release.yml

```
@@ -180,11 +130,12 @@ jobs:
       - name: Create Github Release
         uses: ncipollo/release-action@v1
         with:
-          bodyFile: ${{ env.GITHUB_RELEASE_NOTES }}
+          bodyFile: ${{ env.RELEASE_NOTES_FULLNAME }}
```

Can drop `env.RELEASE_NOTES_FULLNAME` if you change to below:

```suggestion
          bodyFile: ${{ env.RELEASE_NOTES_FILENAME }}.md
```

---

updated